app/views/root/_publication.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <% if publication.published? %>
       <%= link_to "#{Plek.new.website_root}/#{publication.slug}", class: 'link-muted' %>
-    <% elsif publication.safe_to_preview? %>
+    <% elsif !publication.archived? %>
```

Comment:
I reverted the removal of `safe_to_preview?`, but it won't work until that content_models PR is merged, a new gem version released, and a gem version bump in here.